scipy/io/wavfile.py

```diff
@@ -278,10 +297,15 @@ def read(filename, mmap=False):
         else:
             fid.seek(0)
 
-    return fs, data
+    result = [fs, data]
+    if return_cues:
+        result.append(dict(cues))
+    if return_pitch:
+        result.append(pitch)
+    return tuple(result)
 
 
-def write(filename, rate, data):
+def write(filename, rate, data, cues=None, loops=None, bitrate=None):
```

Comment:
"bit_depth" would be more accurate. Bitrate usually refers to bits per second, while bit depth is bits per sample and channel.